ledger2beancount.yml

```diff
@@ -39,6 +52,17 @@ commodity_map:
 altdate_tag: aux-date
 code_tag: code
 
+# Render tags as tags:
+# If true, tags are rendered as #tags after the narration
+# If false, tags are stored as meta data with the key "tags"
+tag-as-tag: true
+
+# A list of regular expressions that will cause a tag to be
+# rendered as a link.
+# For example, if you tag yours trips in the format YYYY-MM-DD-for
+# you could use ^\d\d\d\d-\d\d-\d\d- to render them as links.
+tag-is-link: []
```

Comment:
how about `link_tags` for this? it would highlight that we expect a list here, and it seems more coherent with other configuration options like `payee_tag`, `payer_tag`, etc.